src/librustc_borrowck/borrowck/check_loans.rs

```diff
@@ -755,15 +755,15 @@ impl<'a, 'tcx> CheckLoanCtxt<'a, 'tcx> {
     ///
     /// For example:
     ///
-    /// ```ignore
+    /// ```compile_fail,E0381
     /// let a: i32;
     /// a = 10; // ok, even though a is uninitialized
     ///
     /// struct Point { x: u32, y: u32 }
-    /// let p: Point;
+    /// let mut p: Point;
     /// p.x = 22; // ok, even though `p` is uninitialized
     ///
-    /// let p: Box<Point>;
+    /// let mut p: Box<Point>;
     /// (*p).x = 22; // not ok, p is uninitialized, can't deref
```

Comment:
May be these two examples should be separated into a code block that succeeds (above) and one that fails with E0381 (bellow).